CVE-2020-15906/cvex.yml

```diff
@@ -0,0 +1,5 @@
+blueprint: ubuntu2204
```

Comment:
Use the ubuntu2204-ubuntu2204 blueprint. Install and run the tikiwiki Docker container ubuntu1, copy the exploit to ubuntu2, run the exploit on ubuntu2 that would attack ubuntu1. Make sure that router_raw.pcap captures the network traffic. The goal is to get the traffic!